webpack.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports =
```

Comment:
You should have single webpack.config.js instead three. Please put development and production webpack's configs into one file - webpack.config.js.